### ISSUE TITLE
The sniper printer does not work as expected in a line break context (style issue)

### ISSUE THREAD

**Issue by fermadeiral** [INRIA/spoon#3401]
Hi all,

I have the following code:

```java
        void foo(int x) {
		new IllegalArgumentException("x must be nonnegative");
	}
```

I want to add a `throw` in the constructor call `new IllegalArgumentException`. The transformation works just fine (no exception, the printed code is correct), but a line break is missing, so I get:

```java
        void foo(int x) {throw new java.lang.IllegalArgumentException("x must be nonnegative");
	}
```

Instead of:

```java
        void foo(int x) {
		throw new java.lang.IllegalArgumentException("x must be nonnegative");
	}
```

I am opening a PR with a failing test case to reproduce the issue.

**Comment by nharrand** [comment]
After investigation, it seems that the problem appears for any modified statment in position `0` of a `CtBlock`.

**Comment by monperrus** [comment]
closed per #3400
